templates/etc/fastd/fastd.conf.erb

```diff
@@ -12,7 +12,20 @@ hide mac addresses yes;
 include "secret.conf";
 mtu <%= @mesh_mtu %>; # 1492 - IPv{4,6} Header - fastd Header...
 status socket "/var/run/fastd-status.<%= @mesh_code %>.sock";
-include peers from "peers";
+peer group "nodes" {
+  include peers from "peers";
+<% if @peer_limit %>
+   peer limit <%= @peer_limit %>;
+<% end %>
+}
+peer group "backbone" {
+  include peers from "backbone";
```

Comment:
If you do peer limiting -- and you don't use different fastd instances for gateway-node and gateway-gateway traffic --, you usually want your other gateways to be excluded from that. Happy to move it into the "<% if" if it's too confusing. Forgot to create it, fixed in f19770f09f; if to be filled from git repository, more work would be needed. We don't use a repo for that as of now, as we need to hand-craft the connectivity between gateways (same keys for multiple gateways, must make sure gw01a only connects where gw01b does not directly -- long story).